src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -27,25 +27,43 @@ export default class UniversalDiscoveryModule extends Component {
         super(props);
 
         let selectedLanguage = props.languages.mappings[props.languages.priority[0]];
+        let selectedContentType = {};
+        let contentMeta = null;
+        const isForcedLanguage = props.cotfAllowedLanguages.length === 1 || props.cotfForcedLanguage;
+        const isForcedContentType = props.cotfAllowedContentTypes.length === 1;
+        const isForcedLocation = props.cotfAllowedLocations.length === 1;
 
         this.onLanguageSelected = this.onLanguageSelected.bind(this);
         this.onContentTypeSelected = this.onContentTypeSelected.bind(this);
         this.handlePublish = this.handlePublish.bind(this);
 
         if (props.cotfForcedLanguage) {
+            console.log('[DEPRECATED] cotfForcedLanguage parameter is deprecated');
+            console.log('[DEPRECATED] it will be removed from ezplatform-admin-ui-modules 2.0');
+            console.log('[DEPRECATED] use cotfAllowedLanguages instead');
+
             selectedLanguage = props.languages.mappings[props.cotfForcedLanguage];
         }
 
+        if (props.cotfAllowedLanguages.length === 1) {
```

Comment:
Maybe `isForcedLanguage`?

Reply:
I cannot use `isForcedLanguage` this has forced langue by `allowedLanguages` or `forcedLanguage` because of BC I have to separate those two cases.